### ISSUE TITLE
[Admin

### ISSUE THREAD

**Issue by hkjayakumar** [uwblueprint/website#62]

**Comment by bryanc208** [comment]
I'm trying to understand why this is necessary. We don't leave any feedback/discussion on the admin panel so there is not much sensitive information for anyone with an @blueprint.org email can access. Please correct me if I'm wrong but I feel like this change is not necessary.

**Comment by bryanc208** [comment]
Also, please label as "Feature Request"

**Comment by chrisjluc** [comment]
Access should be restricted for privacy issues. Some of our applicants might not be comfortable with having their information accessible to everybody that's part of our team. We never asked for consent, and best practice is to have PII as restrictive as possible.

**Comment by bryanc208** [comment]
I think the submission of the application is an implicit consent to have the information within the application be made available to be seen to anyone that has an internal stake to Blueprint. Also, what sort of PII will we be restricting? It seems like that information is to be known within Blueprint no matter what once the candidate is fully signed on to be a member of Blueprint. We can't realistically hide name, and email, and academic year/program are not PII. Furthermore, we are realistically not asking for any information from candidates of which they would not feel comfortable sharing on their public profiles. All of this information is publicly available for 99% of candidates through their Facebook/LinkedIn profiles and if not, through the UW Whitepages. 

I think we need to think this through a bit more because it seems like the reasoning is arbitrary at this point. I agree that feedback is something that we should keep private, but being able to see application statuses, and confirming submission of applications should be something every Blueprint member should be allowed to do. Part of my reluctance comes from the fact that I would not appreciate the restrictive nature around the admin panel as a member of Blueprint, especially considering the fact that there is nothing there that requires hiding.

**Comment by chrisjluc** [comment]
Sorry, PII was an accident - I meant personal information.

Well this is definitely a gray area. Some people might have more conservative assumptions with what we're doing with their application data and would feel uncomfortable if everybody can read it. If I was applying and got into Blueprint, I personally would be uncomfortable if anybody could read my application and have assumptions and biases before meeting me. I'd rather tell my story verbally, directly with somebody rather than an application.

I believe people should have the option to tell their story in the way they want to tell it.

But you're right, on the other side we are a transparent organization and people should have access to see who's applying. We're so transparent that I'm happy to say that even people outside of Blueprint can read this thread.

In conclusion, I see both sides of the arguments I think other people should be involved in this conversation and shouldn't be decided by just us two. I'm open to keeping applications open to all members.

**Comment by bryanc208** [comment]
Another area of concern is that we ideally want all members of Blueprint to be equipped to participate in interviews and coffee chats, even though they may not actively participate in the recruiting process all the time. Having access restriction is a strong barrier against this model. 

To address your concern, perhaps the solution is not in the admin panel but to alter the application to encourage lightweight answers and tailor the questions to be as neutral and non-invasive as possible? 

I agree that this is something we should discuss more broadly with the team.

**Comment by hkjayakumar** [comment]
I think @bryanc208 makes some good points, there is not too much info which will not be available publicly.  The only potential areas of concern are the last two questions which are a bit open ended (Tell us a fan fact, and is there anything else you'd like to share)

We should strive to be as open as possible while maintaining candidate privacy. Perhaps this issue can be linked to in slack, and brought up in our next meeting.

Another idea is to delete applications from the admin panel after each recruiting term while archiving that information. This limits the extent to which candidate information is shared.

**Comment by chrisjluc** [comment]
I thought about concerns with openness, we let anybody who's interested in recruiting to be involved in recruiting. In terms of being open about applications for coffee chats we can share it on a need-basis.

My concern with making the answers more lightweight is we're not going to have that depth and understanding of the candidate as much, so this would make it harder to actually filter them.

Good idea, lets bring this up next general if it still hasn't been resolved but for now we should move discussion into slack.
